### ISSUE TITLE
set the path of tools

### ISSUE THREAD

**Issue by firmianay** [Cyber-Buddy/APKHunt#3]
Is it possible to provide a command line parameter, or a configuration file to set the path of jadx and dex2jar, because system packages are usually relatively old, and some distributions cannot be installed directly

**Comment by 0xMagn3t0** [comment]
We are closing this issue as we have addressed the solution and it should work even-if the utilities are being installed anywhere in the Linux.

**Comment by 0xMagn3t0** [comment]
JAR file can be helpful when you want to dig deep into the code level, based on what APKHunt highlights.

**Comment by firmianay** [comment]
But isn't it the same to open the apk with jadx-gui?

**Comment by 0xMagn3t0** [comment]
Yes, it totally depends on the user's choice and what they are comfortable with. One can use JD-GUI to directly open the jar file or Bytecode Viewer for an apk file.
